src/html.js

```diff
@@ -1,15 +1,15 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { PORTAL_TARGETS } from "constants/domNodes";
-import { IS_BUILD } from "constants/env";
+import { IS_PRODUCTION } from "constants/env";
 
 export default function HTML(props) {
   return (
     <html {...props.htmlAttributes}>
       <head>
         <meta charSet="utf-8" />
         <meta httpEquiv="x-ua-compatible" content="ie=edge" />
-        {IS_BUILD && (
+        {IS_PRODUCTION && (
```

Comment:
`IS_BUILD` relied on an environment variable that isn't consistently passed through. Gatsby used it on the server, but not on the client—so our analytics didn't load.